packages/katello/rubygem-pulp_ostree_client/pulp_ostree_client-2.0.0a6.gem

```diff
@@ -0,0 +1 @@
+../../../.git/annex/objects/vV/50/SHA256E-s57856--4ebb1d6ede08752bc85109fdbf572c9cd52766b083b90627becf3e9ecbb1be57.0a6.gem/SHA256E-s57856--4ebb1d6ede08752bc85109fdbf572c9cd52766b083b90627becf3e9ecbb1be57.0a6.gem
```

Comment:
Why is there an additional of the gem, but no removal of the old?

Reply:
because the removal somehow accidentally is already in the repo :woman_shrugging: